setup.py

```diff
@@ -18,61 +18,63 @@
     long_description = fh.read()
 
 tests_require = [
-    'pytest>=4.5.0',
```

Comment:
Is there anything that actually changed here in setup.py?
if not, can we avoid that it shows up in this PR?
It now pollutes the history of this file quite a bit

Reply:
(I can also strip it out when merging the PR, I just want to be sure nothing actually changed)

Reply:
No, there are no real changes anymore in setup.py. The only change left is formatting by black/darker.
So let's keep this file out of the PR.

Initially I made the geopandas test dependency an installation dependency, but I reverted that change.
However that triggered black/darker.

But if we want to switch to black's formatting rules, I think it is better to do that as a single commit (or a PR even) that *only* changes that, so we can easily see that the only change was formatting.


I propose to do a `git checkout master setup.py` to restore the file, but it would still be in the commit history though.
So you may want to purge it from the PR anyway.

Nevertheless, it would be good to restore it to make the CI doublecheck that everything still works.

Reply:
indeed, file-wide formatting fixing commits should not contain anything else.
Which can then be ignored in blame/history views through this "ignore-revs-file" feature (see https://www.git-scm.com/docs/git-blame#Documentation/git-blame.txt---ignore-revs-fileltfilegt and https://docs.github.com/en/repositories/working-with-files/using-files/viewing-a-file#ignore-commits-in-the-blame-view )

I'll purge it when merging the PR using interactive rebase